dp-bulletin-api.nomad

```diff
@@ -0,0 +1,147 @@
+job "dp-bulletin-api" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  update {
+    stagger          = "60s"
+    min_healthy_time = "30s"
+    healthy_deadline = "2m"
+    max_parallel     = 1
+    auto_revert      = true
+  }
+
+  group "web" {
+    count = "{{WEB_TASK_COUNT}}"
+
+    constraint {
+      attribute = "${node.class}"
+      value     = "web"
+    }
+
+    restart {
+      attempts = 3
+      delay    = "15s"
+      interval = "1m"
+      mode     = "delay"
+    }
+
+    task "dp-bulletin-api" {
+      driver = "docker"
+
+      artifact {
+        source = "s3::https://s3-eu-west-1.amazonaws.com/{{DEPLOYMENT_BUCKET}}/dp-bulletin-api/{{REVISION}}.tar.gz"
+      }
+
+      config {
+        command = "${NOMAD_TASK_DIR}/start-task"
+
+        args = ["./dp-bulletin-api"]
+
+        image = "{{ECR_URL}}:concourse-{{REVISION}}"
+
+        port_map {
+          http = 8080
```

Comment:
@janderson2 Could you please clarify this line? This is what was auto-generated but in other services it's set as `"${NOMAD_PORT_http}"` eg.
https://github.com/ONSdigital/dp-search-query/blob/develop/dp-search-query.nomad#L44
https://github.com/ONSdigital/dp-dataset-api/blob/develop/dp-dataset-api.nomad#L44
https://github.com/ONSdigital/dp-search-api/blob/develop/dp-search-api.nomad#L44

Whereas other(s) have a specific port number (eg.  `http = 8080`)…
https://github.com/ONSdigital/dp-frontend-cookie-controller/blob/develop/dp-frontend-cookie-controller.nomad#L44